packages/core/src/contracts/eip20.ts

```diff
@@ -1,18 +1,12 @@
-import { BigNumber } from "bignumber.js";
-import { Observable } from "rxjs";
+import BN from "bignumber.js";
```

Comment:
FYI: You can change `BN` into any name you want, `BigNumber` here would be probably more readable long term.

If you wanna learn more, check `Default exports` section: https://www.typescriptlang.org/docs/handbook/modules.html